pom.xml

```diff
@@ -269,6 +269,34 @@
                 <dateLibrary>java</dateLibrary>
                 <interfaceOnly>true</interfaceOnly>
                 <useSpringBoot3>true</useSpringBoot3>
+                <useTags>true</useTags>
+              </configOptions>
+            </configuration>
+          </execution>
+          <execution>
```

Comment:
Are we going to create a new execution section for each new yaml? Let's try updating the version of openapi-generator-maven-plugin to the latest one and use `inputSpecRootDirectory` instead of `inputSpec`. Theoretically, this should allow us to have just one `execution` for all our yamls.